client/www/pages/dash/index.tsx

```diff
@@ -291,17 +290,24 @@ function Dashboard() {
   const app = apps?.find((a) => a.id === appId);
 
   // ui
-  const availableTabs: TabBarTab[] = tabs
+  const availableTabs: TabItem[] = tabs
     .filter((t) => isTabAvailable(t, app?.user_app_role))
     .map((t) => {
       if (t.id === 'docs') {
         return {
           id: t.id,
           label: t.title,
-          link: app ? `/docs?app=${app.id}` : '/docs',
+          link: {
+            href: app ? `/docs?app=${app.id}` : '/docs',
+            target: '_blank',
+          },
         };
       }
-      return { id: t.id, label: t.title };
+      return {
+        id: t.id,
```

Comment:
note: this _does_ mean that for normal tabs, we no longer call the `nav` function. 

But the only extra thing the nav function did was to save the app id locally. This is not needed for the menu tabs.